src/features/create-token/CreateToken.tsx

```diff
@@ -0,0 +1,116 @@
+//- React Imports
+import React, { FC, useState } from 'react';
+
+//- Style Imports
+import styles from './CreateToken.module.scss';
+
+//- Component Imports
+import { Wizard } from '@zero-tech/zui/src/components/Wizard/Wizard';
+import { CreateTokenBody } from './CreateTokenBody';
+import { CreateTokenHeader } from './CreateTokenHeader/CreateTokenHeader';
+import { ConnectWallet } from 'features/ui/ConnectWallet';
+
+// Lib Imports
+import { useWeb3 } from 'lib/hooks/useWeb3';
+
+//- Type Imports
+import { Step } from '@zero-tech/zui/src/components/StepBar/StepBar.types';
+import { DetailsFormSubmit, TokenomicsFormSubmit } from './CreateToken.types';
+import { MediaType } from '@zero-tech/zui/src/components/MediaInput';
+
+const steps: Step[] = [
+	{
+		id: 'details',
+		title: 'Details',
+	},
+	{
+		id: 'tokenomics',
+		title: 'Tokenomics',
+	},
+	{
+		id: 'launch',
+		title: 'Launch',
+	},
+];
+
+export type CreateTokenProps = {
+	domainName?: string;
+	onClose?: () => void;
+};
+
+export const CreateToken: FC<CreateTokenProps> = ({ domainName, onClose }) => {
```

Comment:
Not sure if I commented this on the last PR, but could be handy to move state + handlers into Context API to save having to drill props through multiple layers.